### PR TITLE
Add risk rosette

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-984, L2B-985

Changes: 
- scaling table layout
- scaling project details header
- scaling project details risk section -> risk analysis
- tooltip popup color
- green (positive) and red (negative) color change
- added risk rosette

Before: 
Screenshot 2023-03-16 at 13 08 32
Screenshot 2023-03-16 at 13 09 59

After: 
Screenshot 2023-03-16 at 13 08 40
Screenshot 2023-03-16 at 13 10 07

### CODE REVIEW

**Comment by antooni** [comment]
visual bug in activity tab
image

**Comment by antooni** [comment]
breakdown icons explainer is no longer needed
image

**Comment by antooni** [comment]
dark mode technology font color does not match the design
image

**Comment by antooni** [comment]
there is a new icon for the tab
image

File: packages/frontend/src/components/rosette/Rosette.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+const args: RiskSentiments = {
+  sequencerFailure: 'bad',
+  stateValidation: 'warning',
+  dataAvailability: undefined,
+  upgradeability: 'bad',
+  validatorFailure: 'warning',
+}
+
+function SmallTemplate(risks: RiskSentiments) {
+  return <SmallRosetteComponent risks={risks} />
```
Comment (antooni) [inline packages/frontend/src/components/rosette/Rosette.stories.tsx:46]: add padding

File: packages/frontend/src/components/rosette/Rosette.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+export const SmallRosette: Story<RiskSentiments> = SmallTemplate.bind({})
+SmallRosette.args = args
+
+function MediumTemplate(risks: RiskSentiments) {
+  return <MediumRosetteComponent risks={risks} />
```
Comment (antooni) [inline packages/frontend/src/components/rosette/Rosette.stories.tsx:53]: add padding

File: packages/frontend/src/components/rosette/Rosette.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+export const MediumRosette: Story<RiskSentiments> = MediumTemplate.bind({})
+MediumRosette.args = args
+
+function BigTemplate(risks: RiskSentiments) {
+  return <BigRosetteComponent risks={risks} />
```
Comment (antooni) [inline packages/frontend/src/components/rosette/Rosette.stories.tsx:60]: add padding

File: packages/frontend/src/components/table/TableView.tsx [hunk 4]
```diff
@@ -90,15 +95,17 @@ export function TableView<T>({ items, columns, rows }: Props<T>) {
                 {columns.map((column, j) => (
                   <td
                     key={j}
-                    className={cx('h-9 md:h-10', column.minimalWidth && 'w-0')}
+                    className={cx('h-9 md:h-14', column.minimalWidth && 'w-0')}
```
Comment (antooni) [inline packages/frontend/src/components/table/TableView.tsx:98]: maybe row can be a bit higher on mobile, or rosetta smaller, or both

File: packages/frontend/src/components/table/TableView.tsx [hunk 5]
```diff
@@ -90,15 +95,17 @@ export function TableView<T>({ items, columns, rows }: Props<T>) {
...
                   >
                     <a
                       href={href}
                       className={cx(
                         'flex h-full w-full items-center',
                         column.alignRight && 'justify-end',
                         column.alignCenter && 'justify-center',
-                        j !== columns.length - 1 && 'pr-3 md:pr-4',
+                        !column.noPaddingRight &&
```
Comment (antooni) [inline packages/frontend/src/components/table/TableView.tsx:106]: tbh this is really unreadable, maybe extract this logic into the function?

File: packages/frontend/src/components/table/TableView.tsx [hunk 6]
```diff
@@ -45,7 +47,10 @@ export function TableView<T>({ items, columns, rows }: Props<T>) {
                 className={cx(
                   'whitespace-pre py-2 text-sm font-medium uppercase text-gray-700 dark:text-gray-600',
                   column.minimalWidth && 'w-0',
-                  i !== columns.length - 1 && 'pr-3 md:pr-4',
+                  !column.noPaddingRight &&
```
Comment (antooni) [inline packages/frontend/src/components/table/TableView.tsx:50]: extract this logic into the function

File: packages/frontend/src/components/rosette/TooltipPopup.tsx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  riskSummary: RiskSentiments
+  riskValues: RiskValues
+}
+
+export function RosetteTooltipPopup({
+  riskValues,
+  riskSummary,
+}: RosetteTooltipProps) {
+  return (
+    <div className="w-88 flex flex-col">
```
Comment (antooni) [inline packages/frontend/src/components/rosette/TooltipPopup.tsx:17]: this w-88 caught my eye, why such an arbitrary value?

**Review by antooni — CHANGES_REQUESTED** [review]
Please fix the visual bugs and extract one part of the logic into the function. Otherwise it looks terrific, great job

File: packages/frontend/src/components/rosette/TooltipPopup.tsx (hunk 7 above)
Comment (michalsidzej) [reply packages/frontend/src/components/rosette/TooltipPopup.tsx:17]: Good catch, this class doesn't even exist

File: packages/frontend/src/components/table/TableView.tsx (hunk 4 above)
Comment (michalsidzej) [reply packages/frontend/src/components/table/TableView.tsx]: I made it smaller, check it out! 
Screenshot 2023-03-09 at 08 52 32

**Review by antooni — APPROVED** [review]

**Review by krzkaczor — COMMENTED** [review]
1. Do we really want to lose numbering on the list :O It looks very odd without any numbers, it's even hard to realize how many solutions do we list...


2. I think padding on the right side is missing here:

image


CC: @kurbanski

**Comment by michalsidzej** [comment]
> 1. Do we really want to lose numbering on the list :O It looks very odd without any numbers, it's even hard to realize how many solutions do we list...
> 2. I think padding on the right side is missing here:
> 
> image
> 
> CC: @kurbanski

The padding was fixed

**Comment by michalsidzej** [comment]
> Have u seen this? Any ideas to improve this screen? @michalsidzej
> 
> image

@krzkaczor Just fixed it. Ready now

**Pull request merged**